### ISSUE TITLE
Wind Direction in N,NNE,NE,ENE,E and so on

### ISSUE THREAD

**Issue by beyondunreal79** [tlskinneriv/awnet_local#35]
Could there be a way to give the option to see the current wind direction in cardinal direction or degrees? 

![Capture2](https://github.com/tlskinneriv/awnet_local/assets/45919008/b1183ea7-614c-4217-b226-31d63d9474b5)

**Comment by tlskinneriv** [comment]
Yep, I can definitely add that as another calculated sensor similar to the Solar Radiation in lux. Will get this in the next release.

**Comment by tlskinneriv** [comment]
Also, if you want a quick config for a template sensor for this:

```yaml
sensor:
  - platform: template
    sensors:
      home_wind_dir_cardinal:
        value_template: >
          {% set direction = ['N','NNE','NE','ENE','E','ESE','SE','SSE','S','SSW','SW','WSW','W','WNW','NW','NNW','N'] %}
          {% set degree = states('<your_winddir_entity_id>')|float(default=0) %}
          {{ direction[((degree+11.25)/22.5)|int(default=0)] }}
```

**Comment by beyondunreal79** [comment]
Your Awesome!
One other quickie thing. Can the last lightning strike be available in miles as well as KM?

**Comment by tlskinneriv** [comment]
Lightning strike distance units unavailable should have been fixed in  [v1.1.3](https://github.com/tlskinneriv/awnet_local/releases/tag/v1.1.3) ref issue #13. If you're already on v1.1.3, but aren't seeing the units option for the sensor, you may need to clear your browser/app cache. If you need both units at the same time, I'd recommend using a template sensor for that.
